lib/phlex/sgml.rb

```diff
@@ -127,7 +127,9 @@ def __final_call__(buffer = +"", context: Phlex::Context.new, view_context: nil,
 		end
 
 		# Output text content. The text will be HTML-escaped.
+		# @param content [String, Symbol, Integer, void] the content to be output on the buffer. Strings, Symbols, and Integers are handled by <code>plain</code> directly, but any object can be handled by overriding <code>format_object</code>
```

Comment:
We can use markdown in these doc strings now that's configured in yardopts so you can use <code>\`</code> fences instead of `<code>`.